dll/win32/shlwapi/precomp.h

```diff
@@ -30,4 +30,6 @@
 
 #include "resource.h"
 
+HRESULT VariantChangeTypeForRead(IN OUT VARIANTARG *pvarg, IN VARTYPE vt);
```

Comment:
I'm not even sure this is needed, because that function appears (so far) to be only used in one single file (ordinal.c) and is defined first before any other functions that use it.

Reply:
This will be used in another C++ file.

Reply:
Deleted.